tests/common/pythonhashseed_test.py

```diff
@@ -0,0 +1,4 @@
+from deep_qa.common.checks import ensure_pythonhashseed_set
+
+def test_pythonhashseed():
+    ensure_pythonhashseed_set()
```

Comment:
I guess assert fails isn't the right thing; I just mean changing the error message inside the test, e.g., by catching the assertion failure and raising with a new message about how you should probably ignore the other failures if this one is failing.